project/src/main/webapp/script.js

```diff
@@ -171,20 +176,38 @@ function createPlaceElement(place) {
     placeElement.appendChild(phone);
     placeElement.appendChild(document.createElement('br'));
   }
+  // Add rating stars
+  addRatingStarsToElement(placeElement, place.rating);
+  placeElement.appendChild(document.createElement('br'));
   return placeElement;
 }
 
-/**
- * Returns a link that represents the given url and shows the given text.
- */
+/** Adds a link that represents the given url and shows the given text. */
 function addLinkToPlaceElement(placeElement, url, linkText) {
-  placeElement.appendChild(document.createElement('br'));
   const link = document.createElement('a');
   link.href = url;
   link.innerText = linkText;
   placeElement.appendChild(link);
 }
 
+/** Adds stars that represent a place's rating. */
+function addRatingStarsToElement(element, rating) {
+  const MAX_RATING = 5;
+  const ratingPercentage = (rating / MAX_RATING) * 100; // Turn to percetage
+  const roundedRating = Math.round(ratingPercentage / 10) * 10; // Round by 10%
+  let outerStars = document.createElement('div');
+  outerStars.classList.add('stars-outer');
+  outerStars.content = "\f006 \f006 \f006 \f006 \f006";
```

Comment:
Add a comment that `\f006` is a star icon?

Reply:
great idea, done